src/Psalm/Internal/Diff/AstDiffer.php

```diff
@@ -23,8 +23,6 @@ class AstDiffer
      * @param array<int, PhpParser\Node\Stmt> $b
      *
      * @return array{0:array<int, array<int, int>>, 1: int, 2: int, 3: array<int, bool>}
-     *
-     * @psalm-pure
```

Comment:
`$is_equal` is not pure 😄

Reply:
( caught by this PR 😃 https://app.circleci.com/pipelines/github/vimeo/psalm/3408/workflows/155fa96e-c151-4f53-9f7d-6f89886902bd/jobs/15663 )